operator/cmd/mesh/manifest-generate_test.go

```diff
@@ -566,13 +566,15 @@ func TestConfigSelectors(t *testing.T) {
 		// First we fetch all the objects for our default install
 		cname := "istiod"
 		deployment := mustFindObject(t, objs, cname, name.DeploymentStr)
+		hpa := mustFindObject(t, objs, cname, name.HPAStr)
```

Comment:
I think you can remove this line as it's not being used anywhere.

Reply:
I'm not sure I understand the overall testing strategy, as the "cross revision" run seems to want to set both a default and a canary and then verify things are not incorrect between them.  The hpa test is already being done in the validateRefererntialIntegrity function but only checking against cname.  I'll remove the extra hpa.

Reply:
/retest